test/build/protobuf.cc

```diff
@@ -33,6 +33,7 @@
 
 #include <google/protobuf/descriptor.h>
 #include <google/protobuf/descriptor.pb.h>
+#include <google/protobuf/stubs/strutil.h>
```

Comment:
I think that's the wrong fix to do. This header seems private and we shouldn't use it. I'm going to check a bit.